Assets/Scripts/UI/InteractTooltip.cs

```diff
@@ -161,16 +165,15 @@ private void OnGUI()
             // Display of text/tooltip that follows the mouse
             if (!hideTooltip && followingMouseCursor && _over)
             {
-                if (textResized)
+                Debug.Log(_tooltipStyle.lineHeight);
```

Comment:
Remove if not needed

Reply:
oh yeah was forgot to remove this line. used for testing some stuff